status-reporter/status_reporter/rabbitmq_reporter.py

```diff
@@ -25,7 +25,7 @@
 from .utils import get_systemd_services, determine_node_status
 
 NODES_API = 'nodes'
-VHOSTS_API = 'vhosts'
+OVERVIEW_API = 'overview'
```

Comment:
You don't have to const everything, if you use it in one place you don't need this